components/SidePillTag.vue

```
@@ -0,0 +1,215 @@
+  '--stroke': props.lineWidth + 'px',
+  '--radius': props.radius + 'px',
+  '--color': props.lineColor,
+}))
+
+function goStore () { window.open(props.storeUrl, '_blank') }
+onMounted(() => { open.value = false })
+</script>
+
+<template>
+  <teleport to="body">
+    <div class="pill" :class="{ open }" :style="vars" role="dialog" aria-live="polite">
```

ESLint

[error] Replace `·class="pill"·:class="{·open·}"·:style="vars"·role="dialog"·aria-live="polite">` with `⏎······class="pill"⏎······:class="{·open·}"⏎······:style="vars"⏎······role="dialog"⏎······aria-live="polite"`

(prettier/prettier)